source/MaterialXTest/Runtime.cpp

```diff
@@ -425,25 +425,29 @@ TEST_CASE("Runtime: NodeGraphs", "[runtime]")
     REQUIRE(port1 == add2.findPort("out"));
 }
 
-TEST_CASE("Runtime: CoreIo", "[runtime]")
+TEST_CASE("Runtime: FileIo", "[runtime]")
 {
     mx::FileSearchPath searchPath;
     searchPath.append(mx::FilePath::getCurrentPath() / mx::FilePath("libraries"));
     {
         // Load in stdlib
         // Create a stage and import the document data.
-        mx::RtObject stageObj = mx::RtStage::createNew("test1");
+        mx::RtObject stageObj = mx::RtStage::createNew("stdlib");
         mx::RtFileIo stageIo(stageObj);
-        stageIo.loadLibraries({ "stdlib" }, searchPath);
+        stageIo.readLibraries({ "stdlib" }, searchPath);
 
         // Get a nodegraph and write a dot file for inspection.
         mx::RtStage stage(stageObj);
+        mx::RtObject graphDef = stage.findElementByName("ND_tiledimage_float");
+        REQUIRE(graphDef);
         mx::RtNodeGraph graph = stage.findElementByName("NG_tiledimage_float");
```

Comment:
Finding the object again is not needed, the RtNodeGraph API can be initialized from the object found above.

Nickpick: `graphDef` could be named `graphObj` to follow code convention used elsewhere in the tests.